### PR TITLE
[5.2] Artisan without mandatory arguments should output usage

### PR DESCRIPTION
**Pull request by riyuk**
PR for Issue #11878

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Maybe this should go to 5.3?

**Comment by GrahamCampbell** [comment]
It is a breaking change to the output.

**Comment by GrahamCampbell** [comment]
Maybe we should output the help after rendering the exception?

**Comment by riyuk** [comment]
@GrahamCampbell that's a good point. moved that if block below the exception.

File: src/Illuminate/Foundation/Console/Kernel.php [hunk 1]
```diff
@@ -109,6 +110,11 @@ public function handle($input, $output = null)
             $this->reportException($e);
 
             $this->renderException($output, $e);
+            if (! $input->hasParameterOption(['--help', '-h'], true) && $input->hasArgument('command')) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/Kernel.php]: Please add a newline above this

File: src/Illuminate/Foundation/Console/Kernel.php [hunk 2]
```diff
@@ -3,6 +3,7 @@
 namespace Illuminate\Foundation\Console;
 
 use Exception;
+use Symfony\Component\Console\Input\ArrayInput;
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/Kernel.php]: Please sort imports by length.

**Comment by GrahamCampbell** [comment]
Please send as single commit to the master branch. :)

**Pull request closed without merging**